common/TelemetryEvents/TelemetryResult.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Windows.DevHome.SDK;
+
+namespace DevHome.Common.TelemetryEvents;
+
+public class TelemetryResult
+{
+    private const int SuccessHResult = 0;
+
+    private const int InvalidArgHResult = unchecked((int)0x80070057);

```

Comment:
Windows.Win32.Foundation.HRESULT.E_INVALIDARG after adding E_INVALIDARG to NativeMethods.txt.